amxmodx/CTask.cpp

```diff
@@ -269,8 +269,10 @@ bool CTaskMngr::taskExists(int iId, AMX *pAmx)
 
 void CTaskMngr::startFrame()
 {
-	for (auto &task : m_Tasks)
+	for(auto i = m_Tasks.length() - 1; i != 0; i--)
```

Comment:
@Arkshine Do you think that they are plugins that rely on the order ? AMX Mod X task system is considered not precise. Also if new tasks inserted we will check them too (but time for task is minimum 0.1).